extras/big-sur/README.md

```diff
@@ -70,8 +70,9 @@ And a special note for MSI Navi users, you no longer require the `ATY,rom`/`-weg
 
 Ensure you've updated to the latest builds (not releases) of OpenCore and all your kexts, as to avoid any odd incompatibility issues. You can find the latest builds of kexts and OpenCore here:
 
-* [Kext Repo](http://kexts.goldfish64.com/)
-* [Driver Repo (contains OpenCore builds too)](http://drivers.goldfish64.com/).
+* [Dortania's Kext & Driver build repo](https://dortania.github.io/builds): updated immediately at every commit, contains DEBUG and RELEASE versions)
+* [Goldfish64 Kext Repo](http://kexts.goldfish64.com/): contains various builds including acidanthera's, but they're only "DEBUG" version
+* [Goldfish64 Driver Repo (contains OpenCore builds too)](http://drivers.goldfish64.com/): same as above, these are only DEBUG builds.
```

Comment:
@Goldfish64's builds are release builds I believe.

Reply:
Ok I checked now, and they seem to be RELEASE by their size. 
If you want you can edit the part where I say that those are DEBUG builds and replace them with only the build-repo which (at least to me) seems more convenient with the drop-down menu to swap across different kexts/drivers and it's a redundant useful source of compiled kexts/drivers